scripts/pre_commit_linter.py

```diff
@@ -317,6 +318,41 @@
 _MESSAGE_TYPE_FAILED = 'FAILED'
 
 
+class FileCache(object):
```

Comment:
Should we make this class non-public, that is, change it to `class _FileCache(object)`?